src/routines/level3/xgemm.cpp

```diff
@@ -33,10 +33,11 @@ Xgemm<T>::Xgemm(Queue &queue, EventPointer event, const std::string &name):
     #include "../../kernels/level3/convert_symmetric.opencl"
     #include "../../kernels/level3/convert_triangular.opencl"
     #include "../../kernels/level3/convert_hermitian.opencl"
+    }, {
+    // keep in mind C1091 in MSVC 2013
```

Comment:
No, AppVeyor is MSVC 2015 (at least how I configured it, not sure if I can also use an older compiler - haven't tried it). I have a specific machine in a corner at home with MSVC 2013, I occasionally test it.